.github/workflows/manual-int-test-workflow.yml

```diff
@@ -0,0 +1,186 @@
+name: Manual Integration Test WorkFlow
+on:
+  workflow_dispatch:
+    inputs:
+      CLUSTER_WIDE:
+        description: 'Run Operator in Cluster Wide Mode. Type false to run cluster in namespace mode'
+        required: false
+        deault: "true"
+jobs:
+  int-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          [
+            appframeworks1,
+            appframeworkc3,
+            appframeworkm4,
+            secret,
+            smartstore,
+            monitoringconsole,
+            scaling,
+            crcrud,
+            licensemanager,
+          ]
+    runs-on: ubuntu-latest
+    env:
+      CLUSTER_NODES: 1
+      CLUSTER_WORKERS: 3
+      SPLUNK_ENTERPRISE_IMAGE: splunk/splunk:edge
+      SPLUNK_OPERATOR_IMAGE_NAME: splunk/splunk-operator
+      SPLUNK_OPERATOR_IMAGE_FILENAME: splunk-operator
+      TEST_FOCUS: "${{ matrix.test }}"
+      # This regex matches any string not containing integration keyword
+      TEST_TO_SKIP: "^(?:[^i]+|i(?:$|[^n]|n(?:$|[^t]|t(?:$|[^e]|e(?:$|[^g]|g(?:$|[^r]|r(?:$|[^a]|a(?:$|[^t]|t(?:$|[^i]|i(?:$|[^o]|o(?:$|[^n])))))))))))*$"
+      TEST_CLUSTER_PLATFORM: eks
+      EKS_VPC_PRIVATE_SUBNET_STRING: ${{ secrets.EKS_VPC_PRIVATE_SUBNET_STRING }}
+      EKS_VPC_PUBLIC_SUBNET_STRING: ${{ secrets.EKS_VPC_PUBLIC_SUBNET_STRING }}
+      TEST_BUCKET: ${{ secrets.TEST_BUCKET }}
+      TEST_INDEXES_S3_BUCKET: ${{ secrets.TEST_INDEXES_S3_BUCKET }}
+      ECR_REPOSITORY: ${{ secrets.ECR_REPOSITORY }}
+      PRIVATE_REGISTRY: ${{ secrets.ECR_REPOSITORY }}
+      S3_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+      ENTERPRISE_LICENSE_LOCATION: ${{ secrets.ENTERPRISE_LICENSE_LOCATION }}
+      CLUSTER_WIDE: ${{ github.event.inputs.release_version }}
```

Comment:
Shouldn't CLUSTER_WIDE be set to the value grabbed on lines 5-8 block? What has release_version to do with it?

Reply:
It was left by mistake. Corrected.